src/renderer/containers/map/infoBox/index.jsx

```diff
@@ -73,6 +80,17 @@ export default class InfoBox extends React.Component {
               <Icon icon="handicap" size="xsm" />
             </div>
           )}
+          <div className="feedback-section">
+            <a
+              className="mail"
+              href={`https://mail.google.com/mail/?view=cm&fs=1&tf=1&to=${
```

Comment:
to add a CC'd email all you need to do is add the following text `&cc=test@test.com`

Reply:
added a cc email